hastory-server/src/Data/Hastory/Server/TestUtils.hs

```diff
@@ -93,14 +94,20 @@ withNewUser clientEnv func = do
     Nothing -> expectationFailure "JWT Cookie not found"
     Just cookie -> func (userId, Token cookie)
 
+-- type AuthCookies = '[ Header "Set-Cookie" SetCookie, Header "Set-Cookie" SetCookie]
 extractJWTCookie :: Headers AuthCookies NoContent -> Maybe B.ByteString
-extractJWTCookie headerList = do
-  (_, cookie) <- find jwtCookie (getHeaders headerList)
-  pure . B.takeWhile (/= ';') $ B.drop 11 cookie
-  where
-    jwtCookie (headerName, headerValue) =
-      headerName == "Set-Cookie" && B.take 11 headerValue == "JWT-Cookie="
+extractJWTCookie headersList =
+  case getHeadersHList headersList of
+    HCons (Header a) _ -> pure (setCookieValue a)
```

Comment:
I think this will only get the first `Set-Cookie` header. That won't work if you have a proxy like cloudflare in front of it.
@prikhi taught me about this.

Reply:
See https://github.com/NorfairKing/intray/pull/7

Reply:
Hi Syd - for my own records, it seems like [RFC 7540](https://tools.ietf.org/html/rfc7540#section-8.1.2.5) allows for two ways to set multiple cookies in the same HTTP response message. You can use multiple `Set-Cookie` headers, or you can use one `Set-Cookie` headers with multiple cookie pairs. To quote from the RFC:

> Therefore, the following two lists of Cookie header fields are semantically equivalent.
> 
> cookie: a=b; c=d; e=f
>
> cookie: a=b
> cookie: c=d
> cookie: e=f

I just wanted to document this bit in case we run into issues later and need to address it.